data/Devanshukoli.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "Devanshu Koli",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "A Recent Front-End Software Intern and Open-source contributor as well. Here to meet with nice people."
+   {
```

Please add the missing links key here to validate your json. 

```suggestion
   "links": [ {
```